bread/views/browse.py

```diff
@@ -59,14 +73,42 @@ def __init__(self, *args, **kwargs):
         self.rowclickaction = kwargs.get("rowclickaction") or self.rowclickaction
         self.backurl = kwargs.get("backurl") or self.backurl
         super().__init__(*args, **kwargs)
+        # set some default bulkactions
+        self.bulkactions = (
+            kwargs.get("bulkactions")
+            or self.bulkactions
+            or [
+                (
+                    Link("excel", label=_("Excel"), icon="download"),
+                    lambda request, qs: export(qs, self.columns),
+                ),
+                (
+                    Link("delete", label=_("Delete"), icon="trash-can"),
+                    delete,
```

Comment:
I thought it was not common in Python to import a function directly.

Reply:
IMHO if it is not more than a few names then I don't see a problem. If we have e.g.

    from mymodule import (
        func_1,
        func_2,
        func_3,
        func_3,
        func_3,
        func_3,
        func_3,